src/Tools/AI Test Toolkit/src/AITTestContextImpl.Codeunit.al

```diff
@@ -155,7 +185,7 @@ codeunit 149043 "AIT Test Context Impl."
         if not IsMultiTurn then
             exit(false);
 
-        if CurrentTurn + 1 > NumberOfTurns then
```

Comment:
This change may cause regression. Have you tested against some of our existing multi-turn scenarios?

Reply:
I can see CurrentTurn is now starts with 1.